### PR TITLE
Added back in experimental OpenLayers support

### PR DESCRIPTION
**Pull request by orangemug**

### CODE REVIEW

**Comment by pathmapper** [comment]
There are new releases from today:

https://github.com/boundlessgeo/ol-mapbox-style/releases/tag/v3.1.0

https://github.com/openlayers/openlayers/releases/tag/v5.2.0

**Comment by orangemug** [comment]
> 2 hours ago

😄 I'll update it

File: src/components/modals/SettingsModal.jsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ class SettingsModal extends React.Component {
           data-wd-key="modal-settings.maputnik:renderer" 
           options={[
             ['mbgljs', 'MapboxGL JS'],
-            // ['ol3', 'Open Layers 3'],
+            ['ol', 'Open Layers 3 (experimental)'],
```
Comment (pathmapper) [inline src/components/modals/SettingsModal.jsx]: Nit: `['ol', 'Open Layers 5 (experimental)'],` or just `['ol', 'Open Layers (experimental)'],`

File: src/components/App.jsx [hunk 2]
```diff
@@ -442,8 +443,10 @@ export default class App extends React.Component {
     let mapElement;
 
     // Check if OL3 code has been loaded?
```
Comment (pathmapper) [inline src/components/App.jsx]: Nit: "OL" instead of "OL3"

**Pull request merged**